addon-test-support/load.js

```diff
@@ -19,4 +22,60 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes("_loadBalance")) {
+    return;
+  }
+  qunit.begin(function() {
+    return new Promise(function(resolve) {
+      Testem.once('next-module-response', function getTestModule(moduleName) {
```

Comment:
incase the `--filter` option is passed, this won't load tests from the module if none of the test name matches the filter. If this happens, it needs to continue emit `get-next-module`

This should look something like:
```
Testem.on('next-module-response', function getTestModule(moduleName) {
        if (moduleName !== undefined) {
          loadIndividualModule(moduleName);
        }

        // if no tests were added, request the next module
        if (QUnit.config.queue.length === 0) {
          Testem.emit('get-next-module');
        } else {
          Testem.removeDuplicateCallbacks('next-module-response', getTestModule);
          resolve();
        }
      });
```